package com.campusdual.classroom;



public class Exercise28 {
    public static void main(String[] args) {

        throw new RuntimeException("Lanzando una excepción desde el método main.");

    }
}
